TelemetryMessage.proto

```diff
@@ -141,12 +141,9 @@ message RelayStatus {
     bool pbv1_open = 3; //pneumatic ball valev state

```

Comment:
this is a legacy typo but might as well change it to "valve state" instead of the valev

Reply:
so this is one is ok to say state huh smh :sleepy:

Reply:
states